### DISCUSSION TITLE
Copy file paths: additional format options

### DISCUSSION THREAD

**Discussion by JayBrown** [mikekazakov/nimble-commander#247] [Ideas]
XtraFinder has some neat options for file path copying; examples:

```
Terminal: /Users/UserName/go/src/github.com/hanwen/go-fuse/all.bash
Short Terminal: ~/go/src/github.com/hanwen/go-fuse/all.bash
URL: file:///Users/UserName/go/src/github.com/hanwen/go-fuse/all.bash
Short URL: file:///~/go/src/github.com/hanwen/go-fuse/all.bash
Windows: \Users\UserName\go\src\github.com\hanwen\go-fuse\all.bash
HFS/AppleScript: VolumeName:Users:UserName:go:src:github.com:hanwen:go-fuse:all.bash
```

Would be great to have in NC. E.g. there could be a submenu for Copy File Paths. Default option (with shortcut ALT-CMD-C) should be the full command line path, as it is currently; the others could be options without shortcuts.

Note: migrated here from the magnumbytes forum

**Comment by singalen** [comment]
Can be done with an external tool.
